### PR TITLE
Make entire calendar event block clickable

### PR DESCRIPTION
**Pull request by sunnyguan**
**Reason for change**
#46 Event not clickable before if not clicking on text, whole calendar event block clickable now. 

**Other changes**
N/A

**Testing**
Local

**Confirm**
- [x] Updated version in manifest.json
- [x] Updated zip file

### CODE REVIEW

**Comment by ez314** [comment]
sunny finally use jetbrains

File: whiteboard/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Whiteboard",
-	"version": "1.3.0",
+	"version": "1.3.1",
```
Comment (ez314) [inline whiteboard/manifest.json:3]: conflict with #45 , gotta change whichever one gets merged second to 1.3.2

Comment (sunnyguan) [reply whiteboard/manifest.json:3]: Oh true, need to fix the whiteboard.zip move too. I'll update this diff after I merge yours

**Review by ez314 — APPROVED** [review]
LGTM! the calendar events are so much easier to click now 🙏

**Pull request merged**